xapian-applications/omega/templates/query

```diff
@@ -153,3 +153,4 @@ $map{$cgilist{B},<input type=hidden name=B value="$html{$_}">}
 </body>
 </html>
 $log{query.log}
+$log{search.log,$hash{md5,$transform{s/^\s+|\s+$$//g,\1,$query}}	$csv{$query,1}	$hitlist{$id$.}	$topdoc}
```

Comment:
Escaping is how CSV allows `,` in cell values. Using `$csv{}` with a non-CSV file format is potentially dangerous, because the escaping rules won't be identical. (Although as noted, the escaping rules for both CSV and TSV are poorly specified.)

Put it this way: since you also control the input processor (in python), you can ensure that any file format works. So it's better to be clear. Using `$csv{}` with TSV is less clear than using it with CSV.